### ISSUE TITLE
Ceph Hammer: Unit ceph-radosgw.service failed to load: No such file or directory.

### ISSUE THREAD

**Issue by bengland2** [ceph/ceph-ansible#642]
This error turns out to be annoying but harmless.

TASK: [ceph-rgw | start rgw on red hat (before or on infernalis)] *************
<gprfc074> REMOTE_MODULE service name=ceph-radosgw state=started
failed: [gprfc074] => {"failed": true}
msg: Failed to start ceph-radosgw.service: Unit ceph-radosgw.service failed to load: No such file or directory.
...

from this rule:

- name: start rgw on red hat (before or on infernalis)
  service:
    name: ceph-radosgw
    state: started
    enabled: yes
  when:
    ansible_os_family == 'RedHat' and
    not is_ceph_infernalis

**Comment by leseb** [comment]
don't we have systemd unit files with hammer?

**Comment by bengland2** [comment]
Hammer doesn't have native systemd support at all, AFAIK.  The "ceph" rpm has /etc/init.d/ceph but does not have systemd files.  If you do "systemctl start ceph" it will work, but that's because systemd has a backwards-compatibility feature with system V init scripts to launch them, I guess.  I found this confusing, but I looked at /usr/lib/systemd/system and saw no systemd-native ceph stuff.  But if you do "systemctl list-dependencies ceph" you'll see that it knows about ceph anyway.
